src/website/app/demos/Flex/index.js

```diff
@@ -45,6 +45,10 @@ powerful and flexible enough to be the foundation of your app's layout.`
           In addition to the props above, FlexItem also accepts all props for{' '}
           <a href="/components/box" key={0}>
             Box
+          </a>{' '}
+          and, if <code key={1}>flex</code> is <code key={2}>true</code>,{' '}
```

Comment:
Since you use "for Box" above, it would be a little more clear with a corresponding "for Flex" here.

Reply:
I tried to make it even more clear, by bringing Flex, the component, before `flex`, the prop: https://github.com/mineral-ui/mineral-ui/blob/f44b50e937a942bcb33651b8891e64536e34a8ba/src/website/app/demos/Flex/index.js#L45-L52